doc/user-guide/04-scenarios.adoc

```diff
@@ -67,6 +67,122 @@ a single confidential command to let the redirection also apply for 'set -x'.
 See the description in /usr/share/rear/conf/default.conf
 how to set variables for secret values in a confidential way.
 
+== Bootable ReaR rescue initramfs on IBM Z (s390x) under z/VM
+
+If you want to create the rescue initramfs with a (tar) backup stored on NFS
+server, add the following settings to `/etc/rear/local.conf`:
+
+[source,bash]
+----
+OUTPUT=RAMDISK
+OUTPUT_URL=file:///path/to/initramfs
+BACKUP=NETFS
+BACKUP_URL=nfs://NFS.server.IP/path/to/backup
+----
+
+Make sure that the path in `OUTPUT_URL` exists and create the rescue initramfs
+and backup and using `rear mkrescue`.
+
+There are two options how to boot the rescue environment on IBM Z (s390x) under
+z/VM created using the `OUTPUT=RAMDISK` setting.
+
+1. Using the `kexec` tool as described in link:../../usr/share/rear/conf/templates/RESULT_usage_RAMDISK.txt[RESULT_usage_RAMDISK.txt],
+   if you have access to a running Linux instance.
+
+2. Using the `zipl` tool for management of boot devices on s390/s390x.
+
+=== Booting ReaR rescue initramfs on IBM Z (s390x) under z/VM with `zipl`
+
+First, we have to select a partition on the disk that we want to make bootable.
+Assume that the disk is called `/dev/dasda` and the partition with the filesystem
+that will contain the kernel and initramfs is `/dev/dasda1`.
+
+We mount this partition to `/mnt/bootdisk` and copy there the kernel and initramfs
+created in the previous section:
+```
+# mount /dev/dasda1 /mnt/bootdisk
+# cp $OUTPUT_URL/HOSTNAME/* /mnt/bootdisk/
+```
+
+NOTE: See `zipl(8)` for all device types supported by the `--target` option
+      (e.g. DASD, SCSI, ...). If you want to use a tape device, see `zipl(8)`
+      to use the `--tape` option instead. Note that the `--tape` option with
+      ReaR is untested so contributions to this guide are welcome! Also, it is
+      not possible to use `zipl` with the z/VM reader device.
+
+IMPORTANT: The following method will overwrite the boot configuration on a device
+           which contains the given `--target` directory, in our case
+           `/dev/dasda`!
+
+Use the following command to make the device mounted at `/mnt/bootdisk` bootable
+with `zipl`:
+```
+# zipl --target=/mnt/bootdisk \
+       --parameters='root=/dev/ram0 ro' \
+       --image=/mnt/bootdisk/kernel-HOSTNAME \
+       --ramdisk=/mnt/bootdisk/initramfs-HOSTNAME.img
+Run /lib/s390-tools//zipl_helper.device-mapper /mnt/bootdisk/
+Building bootmap in '/mnt/bootdisk/'
+Adding IPL section
+  initial ramdisk...: /mnt/bootdisk/initramfs-ibm-z-523.img
+  kernel image......: /mnt/bootdisk/kernel-ibm-z-523
```

Comment:
inconsistency in example - either use HOSTNAME everywhere, or ibm-z-523 everywhere

Reply:
My bad.  Addressed in https://github.com/rear/rear/commit/7f73704142be5e7a88f07bdb0e08fdf3bd7645e0.